archlinuxcn-mirrorlist

```diff
@@ -1,112 +1,128 @@
+## 武昌首义学院 (湖北武汉) (ipv4, ipv6)
 # Server = https://mirrors.wsyu.edu.cn/archlinuxcn/$arch
 
-## 北京交通大学 (北京) (ipv4, ipv6, https)
+## 北京交通大学 (北京) (ipv4, ipv6)
 # Server = https://mirror.bjtu.edu.cn/archlinuxcn/$arch
 
-## 兰州大学 (甘肃兰州) (ipv4, ipv6, http, https)
-# Server = https://mirror.lzu.edu.cn/archlinuxcn/$arch
+## 兰州大学 (甘肃兰州) (ipv4, ipv6)
+# Server = https://mirrors.lzu.edu.cn/archlinuxcn/$arch
 
-## 西安交通大学 (陕西西安) (ipv4, ipv6, http, https)
+## 西安交通大学 (陕西西安) (ipv4, ipv6)
 # Server = https://mirrors.xjtu.edu.cn/archlinuxcn/$arch
 
-## 南阳理工学院 (河南南阳) (ipv4, ipv6, http, https)
+## 南阳理工学院 (河南南阳) (ipv4, ipv6)
 # Server = https://mirror.nyist.edu.cn/archlinuxcn/$arch
 
-## 南京工业大学 (江苏南京) (ipv4, ipv6, http, https)
+## 南京工业大学 (江苏南京) (ipv4, ipv6)
 # Server = https://mirrors.njtech.edu.cn/archlinuxcn/$arch
 
-## 上海科技大学 (上海) (ipv4, https)
+## 上海科技大学 (上海) (ipv4)
 # Server = https://mirrors.shanghaitech.edu.cn/archlinuxcn/$arch
 
-## 中国科学院软件研究所 (北京) (ipv4, https)
+## 中国科学院软件研究所 (北京) (ipv4)
 # Server = https://mirror.iscas.ac.cn/archlinuxcn/$arch
 
-## 北京邮电大学 (北京) (ipv4, ipv6, http, https)
+## 北京邮电大学 (北京) (ipv4, ipv6)
 # Server = https://mirrors.bupt.edu.cn/archlinuxcn/$arch
 
-## NCKU CCNS (Taiwan) (ipv4, http, https)
+## 國立成功大學 (臺灣臺南) (ipv4)
 # Server = https://archlinux.ccns.ncku.edu.tw/archlinuxcn/$arch
 
-## xTom (Hong Kong server) (Hong Kong) (ipv4, ipv6, http, https)
+## xTom (香港伺服器) (香港) (ipv4, ipv6)
+## Blocking users in mainland China
 # Server = https://mirrors.xtom.hk/archlinuxcn/$arch
 
-## xTom (US server) (San Jose, CA, United States) (ipv4, ipv6, http, https)
+## xTom (United States server) (San Jose, United States) (ipv4, ipv6)
+## Blocking users in mainland China
 # Server = https://mirrors.xtom.us/archlinuxcn/$arch
 
-## xTom (Netherlands server) (Amsterdam, the Netherlands) (ipv4, ipv6, http, https)
+## xTom (Netherlands server) (Amsterdam, Netherlands) (ipv4, ipv6)
 # Server = https://mirrors.xtom.nl/archlinuxcn/$arch
 
-## xTom (Germany server) (Duesseldorf, Germany) (ipv4, ipv6, http, https)
+## xTom (Germany server) (Dusseldorf, Germany) (ipv4, ipv6)
 # Server = https://mirrors.xtom.de/archlinuxcn/$arch
 
-## xTom (Estonia server) (Tallinn, Estonia) (ipv4, ipv6, http, https)
+## xTom (Estonia server) (Tallinn, Estonia) (ipv4, ipv6)
+## Blocking users in mainland China
 # Server = https://mirrors.xtom.ee/archlinuxcn/$arch
 
-## xTom (Japan server) (Osaka, Japan) (ipv4, ipv6, http, https)
+## xTom (Japan server) (Osaka, Japan) (ipv4, ipv6)
+## Blocking users in mainland China
 # Server = https://mirrors.xtom.jp/archlinuxcn/$arch
 
-## xTom (Australia server) (Sydney, Australia) (ipv4, ipv6, http, https)
+## xTom (Australia server) (Sydney, Australia) (ipv4, ipv6)
+## Blocking users in mainland China
 # Server = https://mirrors.xtom.au/archlinuxcn/$arch
 
-## Open Computing Facility, UC Berkeley (Berkeley, CA, United States) (ipv4, ipv6, http, https)
+## University of California, Berkeley (Berkeley, United States) (ipv4, ipv6)
```

Comment:
Why do you remove the state abbreviation? There are a lot of places named "Berkeley" in the US.